google/cloud/security/inventory/pipelines/load_group_members_pipeline.py

```diff
@@ -102,16 +104,37 @@ def _retrieve(self):
                 raise inventory_errors.LoadDataPipelineError(e)
 
             group_members_map.append((group_id, group_members))
+            LOGGER.debug('Retrieved members from {0}: {1}'.format(
```

Comment:
nit: Google-style preferred way here is using string substitution, e.g. ...members from %s, %d', string, integer)